docs/phylum_group_list.md

```diff
@@ -0,0 +1,23 @@
+---
+title: phylum group list
+category: 6255e67693d5200013b1fa3e
+hidden: false
+---
+List all groups the user is part of
```

Comment:
Nit: should it be "groups that the user is a member of" or something like that? I don't know if we normally say users are part of groups or members of groups.

Reply:
I just copied what the CLI help had, so if we want to change it here we should change it there as well

Reply:
Yeah I'd definitely advocate to keep the docs in line with the CLI.

Reply:
update in docs and CLI